standardtracer/standardcontext.go

```diff
@@ -0,0 +1,48 @@
+package standardtracer
+
+import "sync"
+
+type StandardContext struct {
```

Comment:
(I *do* think there's value in a util package that various tracer impls can reuse...)